### PR TITLE
Conditional 4i

### PR DESCRIPTION
**Pull request by DriessenA**
Extra minor changes to make 4i experiments run smoothly and be compatible with older configs

### CODE REVIEW

File: cmonge/models/embedding.py [hunk 1]
```diff
@@ -43,12 +44,25 @@ def __init__(
...
         model_dir: str,
-        datamodule: ConditionalDataModule,  # For compatability
+        datamodule: ConditionalDataModule,
+        dose_split: bool = True,
+        drug_list: List = [
+            "abexinostat",
+            "belinostat",
+            "dacinostat",
+            "entinostat",
+            "givinostat",
```
Comment (jannisborn) [inline cmonge/models/embedding.py]: let's avoid hard coding dataset specific content

Comment (jannisborn) [reply cmonge/models/embedding.py]: rather update the configs

**Comment by jannisborn** [comment]
@DriessenA please
- Rebase this to `main` since it shows a lot of traces that we already merged in #13. In the current state it's hard to discern what is even the content of this PR
- Resolve merge conflict

**Comment by DriessenA** [comment]
Okay I think the rebasing worked :p

**Review by jannisborn — APPROVED** [review]
LGTM 👍🏼

**Comment by DriessenA** [comment]
Delete the branch as well already?

**Pull request merged**